socorro/cron/crontabber_app.py

```diff
@@ -3,67 +3,109 @@
+)
 
-# this class is for eventual support of CronTabber with the universal
-# socorro app.
 from socorro.app.socorro_app import App
 
 
 # NOTE(willkg): This is what we have in -prod
-DEFAULT_JOBS = '''
-socorro.cron.jobs.weekly_reports_partitions.WeeklyReportsPartitionsCronApp|7d
-socorro.cron.jobs.matviews.ProductVersionsCronApp|1d|05:00
-socorro.cron.jobs.matviews.SignaturesCronApp|1d|05:00
-socorro.cron.jobs.matviews.RawUpdateChannelCronApp|1d|05:00
-socorro.cron.jobs.matviews.ADUCronApp|1d|08:30
-socorro.cron.jobs.matviews.DuplicatesCronApp|1h
-socorro.cron.jobs.matviews.ReportsCleanCronApp|1h
-socorro.cron.jobs.bugzilla.BugzillaCronApp|1h
-socorro.cron.jobs.matviews.BuildADUCronApp|1d|08:30
-socorro.cron.jobs.matviews.AndroidDevicesCronApp|1d|05:00
-socorro.cron.jobs.matviews.GraphicsDeviceCronApp|1d|05:00
-socorro.cron.jobs.ftpscraper.FTPScraperCronApp|1h
-socorro.cron.jobs.elasticsearch_cleanup.ElasticsearchCleanupCronApp|7d
-socorro.cron.jobs.drop_old_partitions.DropOldPartitionsCronApp|7d
-socorro.cron.jobs.truncate_partitions.TruncatePartitionsCronApp|7d
-socorro.cron.jobs.clean_raw_adi_logs.CleanRawADILogsCronApp|1d
-socorro.cron.jobs.clean_raw_adi.CleanRawADICronApp|1d
-socorro.cron.jobs.clean_missing_symbols.CleanMissingSymbolsCronApp|1d
-socorro.cron.jobs.missingsymbols.MissingSymbolsCronApp|1d
-socorro.cron.jobs.featured_versions_automatic.FeaturedVersionsAutomaticCronApp|1h
-socorro.cron.jobs.upload_crash_report_json_schema.UploadCrashReportJSONSchemaCronApp|1h
-'''
+DEFAULT_JOBS_BASE = [
+    'socorro.cron.jobs.weekly_reports_partitions.WeeklyReportsPartitionsCronApp|7d',
+    'socorro.cron.jobs.matviews.ProductVersionsCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.SignaturesCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.RawUpdateChannelCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.ADUCronApp|1d|08:30',
+    'socorro.cron.jobs.matviews.DuplicatesCronApp|1h',
+    'socorro.cron.jobs.matviews.ReportsCleanCronApp|1h',
+    'socorro.cron.jobs.bugzilla.BugzillaCronApp|1h',
+    'socorro.cron.jobs.matviews.BuildADUCronApp|1d|08:30',
+    'socorro.cron.jobs.matviews.AndroidDevicesCronApp|1d|05:00',
+    'socorro.cron.jobs.matviews.GraphicsDeviceCronApp|1d|05:00',
+    'socorro.cron.jobs.ftpscraper.FTPScraperCronApp|1h',
+    'socorro.cron.jobs.elasticsearch_cleanup.ElasticsearchCleanupCronApp|7d',
+    'socorro.cron.jobs.drop_old_partitions.DropOldPartitionsCronApp|7d',
+    'socorro.cron.jobs.truncate_partitions.TruncatePartitionsCronApp|7d',
+    'socorro.cron.jobs.clean_raw_adi_logs.CleanRawADILogsCronApp|1d',
+    'socorro.cron.jobs.clean_raw_adi.CleanRawADICronApp|1d',
+    'socorro.cron.jobs.clean_missing_symbols.CleanMissingSymbolsCronApp|1d',
+    'socorro.cron.jobs.missingsymbols.MissingSymbolsCronApp|1d',
+    'socorro.cron.jobs.featured_versions_automatic.FeaturedVersionsAutomaticCronApp|1h',
+    'socorro.cron.jobs.upload_crash_report_json_schema.UploadCrashReportJSONSchemaCronApp|1h',
+]
```

Comment:
I just converted this from one big `\n` delimited string into a Python list of strings. That's easier to manipulate so we can do things like "all the default jobs plus this one".